src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/authentication/GithubAuthenticator.kt

```diff
@@ -86,23 +86,11 @@ class GithubAuthenticator(private val userRepository: UserRepository,
         }
     }
 
-    private fun userBelongToOrg(githubOrg: String, user: User): Boolean
+    private fun currentUserBelongsToOrg(githubOrg: String): Boolean
```

Comment:
This was probably not the cause of the original bug, but I was finding that using this route through the Organization service was not returning myself as a member of vimc, even though the docs state that it should return non-public members. This alternative, getting the orgs for the authenticated user does work. (This bit was probably working for Rich  before as he is a public member of vimc.)